ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -211,28 +206,31 @@ export class ScenePlayerImpl extends React.Component<
     };
 
     const seekHook = (seekToPosition: number, _videoTag: HTMLVideoElement) => {
-      if (
-        !_videoTag.src ||
-        ScenePlayerImpl.isDirectStream(_videoTag.src) ||
-        _videoTag.src.endsWith(".m3u8")
-      ) {
-        // direct stream - fall back to default
+      if (!_videoTag.src || _videoTag.src.endsWith(".m3u8")) {
         return false;
       }
 
-      // remove the start parameter
-      let { src } = _videoTag;
+      if (ScenePlayerImpl.isDirectStream(_videoTag.src)) {
+        if (_videoTag.dataset.start) {
+          /* eslint-disable no-param-reassign */
+          _videoTag.dataset.start = "0";
+          /* eslint-enable no-param-reassign */
```

Comment:
You  can use `// eslint-disable-next-line no-param-reassign` rather than disabling and enabling.

Reply:
Fixed. Sorry i'm pretty new to modern frontend dev. My work world is full of hacky jquery.